store/sqlstore/team_store.go

```diff
@@ -472,30 +544,30 @@ func (s SqlTeamStore) teamSearchQuery(opts *model.TeamSearch, countQuery bool) s
 // SearchAll returns from the database a list of teams that match the Name or DisplayName
 // passed as the term search parameter.
 func (s SqlTeamStore) SearchAll(opts *model.TeamSearch) ([]*model.Team, error) {
-	var teams []*model.Team
+	var teams TeamWithPolicyIdList
```

Comment:
Can you explain why do we need this? Since it wasn't there originally, I wouldn't expect us to need it now.

Reply:
PolicyID from Team model has db:"-" tag. It wokred fine with gorp variant. 
Yet sqlx fail join query(_missing destination name_). I used fileInfoWithChannelID  as an example. 
Is there better way to avoid that error?

Reply:
Ah yes, you are right. The current way is fine. Please fix the tests, and we will take another look. Thank you!

Reply:
@krotovkk - This shouldn't be needed now as our migrations have moved to the new model. You can remove the `db:"-"` tag from the model structs.

Reply:
Is migration needed for policyId?

Reply:
The migrations are decoupled from the model structs now. There's no migration needed for this.